src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -458,6 +470,29 @@ async def test_error_without_raise_in_unary_unary(self):
         rpc_error = exception_context.exception
         self.assertEqual(grpc.StatusCode.INTERNAL, rpc_error.code())
 
+    async def test_error_without_raise_in_stream_unary(self):
+
+        async def request_iterator():
+            for _ in range(_NUM_STREAM_REQUESTS):
+                yield _REQUEST
+
+        call = self._channel.stream_unary(
+            _ERROR_WITHOUT_RAISE_IN_STREAM_UNARY)(request_iterator())
+
+        self.assertEqual(grpc.StatusCode.INTERNAL, await call.code())
+        self.assertFalse(call.cancelled())
+        self.assertTrue(call._async_request_poller.done())
```

Comment:
this assertion doesn't actually fail w/o the changes in _call - but calling "cancelled()" seemed a little "on the nose" to just go grabbing the internal attribute directly to assert on what should be an implimentation detail.